torch/_dynamo/convert_frame.py

```diff
@@ -1193,6 +1193,9 @@ def format_func_info(code: CodeType) -> str:
                 "has_guarded_code": guarded_code is not None,
                 "config_suppress_errors": config.suppress_errors,
                 "config_inline_inbuilt_nn_modules": config.inline_inbuilt_nn_modules,
+                # Note: should we be adding install_params here too? or since only really used in
```

Comment:
Would this be helpful to log in this metrics for debugability? Or okay to skip since this should really only be used for export?

Reply:
We can ignore this in this PR. Just to ensure that the logging part is a separate PR and can be reverted in isolation if necessary.